vdr/src/contracts/cl/types/schema.rs

```diff
@@ -17,6 +17,55 @@ pub struct Schema {
     pub attr_names: Vec<String>,
 }
 
+impl Schema {
+    pub fn require_valid_id(&self, expected_schema_id: &SchemaId) -> VdrResult<()> {
```

Comment:
let add method just returning`id` for schema and move comparison

Reply:
Added